Libraries/MSSQLDevice/Source/Device.MSSQL/MSSQLTypes.cs

```diff
@@ -446,11 +450,83 @@ public class MSSQLBinary : SQLBinary
 
 		protected override string InternalNativeDomainName(MetaData AMetaData)
 		{
-			return "image";
+			return "varbinary(max)";

```

Comment:
Is this change to MSSQLBinary necessary to support the Geography changes? I'm concerned about changing the native type due to backwards compatibility. What happens to a system that has MSSQLBinary data that is already declared to be of type "image" in the underlying system?